limitador/src/storage/redis/redis_cached.rs

```diff
@@ -138,7 +138,9 @@ impl AsyncCounterStorage for CachedRedisStorage {
                         counter_ttls_msecs[i],
                         ttl_margin,
                     );
-                    let remaining = counter_vals[i].unwrap_or(counter.max_value()) - delta;
+                    let remaining = counter.max_value()
+                        - counter_vals[i].unwrap_or(counter.max_value())
```

Comment:
Shouldn't be `0` ??
`counter_vals[i].unwrap_or(0)` ??

Like I set in https://github.com/Kuadrant/limitador/blob/main/limitador/src/storage/redis/redis_async.rs#L123

Reply:
Yes!